site/specs-source/voice.json

```diff
@@ -3012,247 +3075,298 @@
           },
           "tag": {
             "nullable": true,
-            "type": "string"
+            "type": "string",
+            "description": "The custom string attached to the conference that will be sent with callbacks."
           },
           "activeMembers": {
             "type": "array",
+            "nullable": true,
             "items": {
               "$ref": "#/components/schemas/ConferenceMemberState"
-            }
+            },
+            "description": "A list of active members of the conference. Omitted if this is a response to the [Get Conferences endpoint](/apis/voice/#operation/getConferences)"
           }
         }
       },
       "ConferenceMemberState": {
         "type": "object",
         "properties": {
           "callId": {
-            "type": "string"
+            "type": "string",
+            "description": "The call ID of this conference member"
           },
           "conferenceId": {
-            "type": "string"
+            "type": "string",
+            "description": "The Bandwidth-generated conference ID"
           },
           "memberUrl": {
-            "type": "string"
+            "type": "string",
+            "description": "A URL that may be used to retrieve information about or update the state of this conference member. This is the URL of this member's [Get Conference Member](/apis/voice/#operation/getConferenceMember) endpoint and [Modify Conference Member](/apis/voice/#operation/modifyConferenceMember) endpoint."
           },
           "mute": {
-            "type": "boolean"
+            "type": "boolean",
+            "description": "Whether or not this member is currently muted. Members who are muted are still able to hear other participants.\n\nIf used in a PUT request, updates this member's mute status. Has no effect if omitted."
           },
           "hold": {
-            "type": "boolean"
+            "type": "boolean",
+            "description": "Whether or not this member is currently on hold. Members who are on hold are not able to hear or speak in the conference.\n\nIf used in a PUT request, updates this member's hold status. Has no effect if omitted."
           },
           "callIdsToCoach": {
             "nullable": true,
             "type": "array",
             "items": {
               "type": "string"
-            }
+            },
+            "description": "If this member had a value set for `callIdsToCoach` in its [Conference](/docs/voice/bxml/conference) verb or this list was added with a previous PUT request to modify the member, this is that list of calls.\n\nIf present in a PUT request, modify's the calls that this member is coaching. Has no effect if omitted. See the documentation for the [Conference](/docs/voice/bxml/conference) verb for more details about coaching.\nNote that this will not add the matching calls to the conference; each call must individually execute a Conference verb to join."
```

Comment:
```suggestion
            "description": "If this member had a value set for `callIdsToCoach` in its [Conference](/docs/voice/bxml/conference) verb or this list was added with a previous PUT request to modify the member, this is that list of calls.\n\nIf present in a PUT request, modifies the calls that this member is coaching. Has no effect if omitted. See the documentation for the [Conference](/docs/voice/bxml/conference) verb for more details about coaching.\nNote that this will not add the matching calls to the conference; each call must individually execute a Conference verb to join."
```